massgov/mayflower
# DP-8086: Mayflower React: Port General Teaser Molecule from Mass.Gov Search.

## Description
This adds the following new components: GeneralTeaser, IconLink, Link. This also modifies the Image component to add a new prop for classnames: classes.

Backstop configuration has been updated to support creating a custom override for component stories.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-8086)

## Steps to Test


1. Pull the branch and run `npm install`.
2. Go to http://localhost:6006/. Click on Organisms -> GeneralTeaser. Verify that the GeneralTeaser's look/feel is okay and all knobs work.
3. Repeat Step two with IconLink and Link (under Molecules).

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.
